packages/list/test/mwc-list.test.ts

```diff
@@ -1444,6 +1444,8 @@ suite('mwc-list:', () => {
         await items[1].updateComplete;
         await element.updateComplete;
 
+        await new Promise((res) => setTimeout(res, 50));
```

Comment:
Does this pass now if you remove this? if not, does it pass if you await itemsReady?

Reply:
Yes it does. Forgot to remove it - now it is removed.